v3/internal/logger/logger.go

```diff
@@ -72,8 +73,11 @@ func (f *logFile) fire(level, msg string, ctx map[string]interface{}) {
 		msg,
 		ctx,
 	})
-	if nil == err {
-		f.l.Print(string(js))
+	if err == nil {
+		// scrub license keys from any portion of the log message
+		re := regexp.MustCompile(`license_key=[a-fA-F0-9.]+`)
+		sanitized := re.ReplaceAllLiteralString(string(js), "license_key=[redacted]")
+		f.l.Print(sanitized)
 	} else {
 		f.l.Printf("unable to marshal log entry: %v", err)
```

Comment:
Would it be helpful to have a check outside as well on the off-chance that our key is printed